database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
```

Comment:
SERIAL is an integer. We also could use integers for the subscription IDs of cause, but then you have a sequence generator in the database.